### ISSUE TITLE
Support currency conversion rates

### ISSUE THREAD

**Issue by JJRcop** [mr-kenhoff/qalculate-android#29]
Qalculate! supports fetching currency rates from the internet and permitting queries like `1 USD to EUR`, or even `1$/gallon to €/liter`. It would be nice to have this function in this mobile app.

**Comment by mr-kenhoff** [comment]
Totally agree! For that, libcurl has to be built and support for fetching the exchange rates must be enabled when building libqalculate.
I created an issue for that in libqalculate-android: https://github.com/mr-kenhoff/libqalculate-android/issues/4

**Comment by JJRcop** [comment]
This is technically separate from including libcurl, but if possible I'd make sure the `exrates`command works so you can refresh on demand

**Comment by boehs** [comment]
In the interest of collaboration, here is the code in use by my own app: maculate: https://github.com/toastclub/maculate/blob/f3e6f840510c149fa1fea6f817a32842829687e8/libqalc/Qalculate/main.cpp#L116-L164

I know how hard it is to read the code of qalculate sometimes so I thought this could be a good starting point.

One oddity I've not figured out yet is that bitcoin is inversed but nothing else is as far as I can tell

**Comment by redfast00** [comment]
The README of this project does confusingly mention currency conversion in the second sentence, so I thought it was my fault that I couldn't get it to work.

@jherkenhoff is it maybe an idea to remove currency conversion from the README until it's implemented?

**Comment by jherkenhoff** [comment]
You are right, that is missleading. I will work on this feature right away ;)

**Comment by redfast00** [comment]
Oh wow, thank you! That's even beter <3

**Comment by jherkenhoff** [comment]
Early support for currencies and currency conversion is now implemented 👍 
Symbols (e.g. using $ for US dollars) are not yet working, but that should be doable...

<img width="50%" alt="Image" src="https://github.com/user-attachments/assets/6eef6ec0-6227-4ddb-bb62-2e00c9f8492d" />

**Comment by jherkenhoff** [comment]
Fixed in https://github.com/jherkenhoff/qalculate-android/commit/0064818b383bfdacb2ac4a4350616bdec2754620
